pyproject.toml

```diff
@@ -38,7 +38,7 @@ select = ["E", "W", "F", "C", "N", "R", "D", "H"]
 # Ignore D107 Missing docstring in __init__
 ignore = ["W503", "E501", "D107"]
 # D100, D101, D102, D103: Ignore missing docstrings in tests
-per-file-ignores = ["tests/*:D100,D101,D102,D103,D104"]
+per-file-ignores = ["tests/*:D100,D101,D102,D103,D104,D205,D212,D415,DC0020"]
```

Comment:
I'm trying to figure it out why D205 is needed here but in [Indico](https://github.com/canonical/indico-operator/blob/main/pyproject.toml#L45) is not even with the same kind of docstring for tests. Any ideas?

Reply:
According to the [contributing guide's example](https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#test-structure) there is no line after the summary line (that is empty, of course)

Reply:
Just a followup: Now I see it was because of the .flake8 file. Indico still has it.